lib/core/src/display/camera/camera2d.rs

```diff
@@ -144,27 +184,57 @@ impl Camera2dData {
         let transform_dirty        = TransformDirty2::new(logger.sub("transform_dirty"),());
         let transform_dirty_copy   = transform_dirty.clone();
         let transform              = DisplayObjectData::new(logger);
-        let zoom_uniform           = globals.add_or_panic("zoom",1.0);
+        let zoom_update_callback   = None;
         transform.set_on_updated(move |_| { transform_dirty_copy.set(); });
         transform.mod_position(|p| p.z = 1.0);
         projection_dirty.set();
-        Self {transform,screen,projection,clipping,alignment,zoom,zoom_uniform,native_z,view_matrix
-             ,projection_matrix,view_projection_matrix,projection_dirty,transform_dirty}
+        let mut camera = Self {
+            transform,
+            screen,
+            projection,
+            clipping,
+            alignment,
+            zoom,
+            zoom_update_callback,
+            native_z,
+            view_matrix,
+            projection_matrix,
+            view_projection_matrix,
+            projection_dirty,
+            transform_dirty
```

Comment:
Why this change? It takes sooo much space now. Just keep it in 2 lines please :)

Reply:
It's 3 lines now :P But yup